### PR TITLE
Make normalize optional in openeye.get_charges

### PR DESCRIPTION
**Pull request by bas-rustenburg**
Added an optional keyword argument, and a unit test that sets it to False.

### CODE REVIEW

**Comment by bas-rustenburg** [comment]
Is there a specific way I can run (just the) openeye nosetests on my local computer?

This doesn't seem to work:
```
nosetests -vv --exe openmoltools/tests/test_openeye.py
nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
Failure: NameError (name 'openeye_exception_message' is not defined) ... ERROR

======================================================================
ERROR: Failure: NameError (name 'openeye_exception_message' is not defined)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/bas/miniconda3/envs/py2k/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/bas/miniconda3/envs/py2k/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/bas/miniconda3/envs/py2k/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/bas/Repos/openmoltools/openmoltools/tests/test_openeye.py", line 37, in <module>
    @skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
NameError: name 'openeye_exception_message' is not defined

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)

```

File: openmoltools/tests/test_openeye.py [hunk 1]
```diff
@@ -42,6 +42,15 @@ def test_butanol_keepconfs():
     assert m1.NumConfs() == 1, "This OEMol was created to have a single conformation."
     assert m1.NumAtoms() == 15, "Butanol should have 15 atoms"
 
+@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
```
Comment (bas-rustenburg) [inline openmoltools/tests/test_openeye.py]: I think this line is the reason my nosetests don't run, but I just copied it from test_butanol_keepconfs.

**Comment by bas-rustenburg** [comment]
Added a little fix to the nosetests. Maybe these were broken all along, but no one ever ran them because of travis skipping them?

**Comment by jchodera** [comment]
Try `nosetests -vv --nocapture --with-doctest openmoltools.tests.test_openeye`

**Comment by bas-rustenburg** [comment]
One fail, but not something caused by my PR. 
```
FAIL: openmoltools.tests.test_openeye.test_butanol
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/bas/miniconda3/envs/py2k/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/bas/Repos/openmoltools/openmoltools/tests/test_openeye.py", line 61, in test_butanol
    assert m1.NumConfs() >= 2, "Butanol should have multiple conformers."
AssertionError: Butanol should have multiple conformers.
```

**Comment by bas-rustenburg** [comment]
No clue what is causing the particular error, but this PR is good for merging as far as the added feature is concerned.

**Comment by jchodera** [comment]
That failure is because the default behavior of `get_charges()` was changed in https://github.com/choderalab/openmoltools/pull/147 but the test was not changed appropriately.  I'm not sure how that happened.

Can you add the argument `keep_confs=-1` to the call to `get_charges()` here?
https://github.com/bas-rustenburg/openmoltools/blob/master/openmoltools/tests/test_openeye.py#L59

**Comment by bas-rustenburg** [comment]
Done.

**Comment by jchodera** [comment]
Woohoo!  Thanks!

**Comment by davidlmobley** [comment]
Yes, thanks! And thanks for catching the issue with the test. I probably
botched that when making a change before we had Travis testing the OpenEye
stuff, and apparently Kyle didn't notice when checking, either.

On Wed, Oct 28, 2015 at 3:39 PM, John Chodera <notifications@github.com>
wrote:

> Woohoo! Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/choderalab/openmoltools/pull/159#issuecomment-152019016>
> .
>



-- 
David Mobley
Associate Professor
Department of Pharmaceutical Sciences
Department of Chemistry
3134B Natural Sciences I
University of California, Irvine
Irvine, CA 92697
dmobley@uci.edu
work (949) 824-6383
cell (949) 385-2436

**Pull request merged**